src/jira-operations.js

```diff
@@ -61,15 +71,21 @@ export let findParent = _.memoize(
           return jiraClient.findIssue(parentKey);
         }
       }
-
       return getEpicLinkField(jiraClient)
-        .then(linkField => jiraClient.findIssue(issue.fields[linkField]));
-
+        .then(linkField =>
+          jiraClient.findIssue(issue.fields[linkField])
+            .catch(function(){
+                const subtaskParentKey = findIssueLinkParentKeyByType(issue, 'Sub-task');
+                return subtaskParentKey ? jiraClient.findIssue(subtaskParentKey) : Promise.resolve(undefined);
+            })
+        );
```

Comment:
This can be simplified.
```js
.then(linkfield => jiraClient.findIssue(issue.fields[linkfield]))
.catch(() => {
  /* do the things */
});
```
Less arrowheading and easier to read. It is also more concise to use an arrow function here.